apps/dotcom/client/e2e/tests/homepage.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { expect, test } from '../fixtures/tla-test'
+
+test.beforeEach(async ({ homePage }) => {
+	await homePage.goto()
+	await homePage.isLoaded()
+})
+
+test('can toggle sidebar', async ({ editor, sidebar }) => {
+	await editor.ensureSidebarClosed()
+	await expect(sidebar.sidebarLogo).not.toBeVisible()
+	await editor.toggleSidebar()
+	await expect(sidebar.sidebarLogo).toBeVisible()
+})
+
+test('can create new document', async ({ page, editor, sidebar }) => {
+	await editor.ensureSidebarOpen()
```

Comment:
i think my only main feedback is it feels like we're going to need open up the sidebar for most, or at least a lot of tests. feels kind of annoying to have to do that every time.

i guess it's fine for now but i do wonder if we should default it to being open on e2e tests, not sure.

Reply:
Yeah. We can also add a helper to the `HomePage` fixture: `gotoAndOpenSiderbar`. You need to go to some url to start the test anyway so combining it with opening the sidebar might be good enough.